File: CrazyBooks_Final/CrazyBooks.Lib/Models/Lend.cs
using Newtonsoft.Json;
using System;

namespace CrazyBooks.Lib.Models
{
    public class Lend : Entity
    {
        public DateTime? LendedOn { get; set; }
        public DateTime? ReturnDate { get; set; }
        public Guid UserId { get; set; }

        [JsonIgnore]
        public User Users { get; set; }

        public Guid BookId { get; set; }
        [JsonIgnore]
        public Book Books { get; set; }

    }
}

File: CrazyBooks_Final/CrazyBooks.Web/wwwroot/Scripts/Models/Lend.js
class Lend extends Entity
{
    constructor(json)
    {
        super(json);

        if (json) {
            this.BookId = json.bookId;
            this.Books = json.bookId;
            this.UserId = json.userId;
            this.Users = json.bookId;
            this.LendedOn = json.lendedOn;
            this.ReturnDate = json.returnDate;
        }
        else {
            this.BookId = "";
            this.Books = "";
            this.UserId = "";
            this.Users = "";
            this.LendedOn = "";
            this.ReturnDate = "";
        }

        
        //this.LendedOn = new Date();
        //this.ReturnDate = new Date();

        //this.LendedOn.toLocaleString();
        //this.ReturnDate.toLocaleString();

        //this.ReturnDate.setMonth(this.ExpiresOn.getMonth() + 1);
    }
}